HousingSearchListener/V1/Domain/Tenure/TempAccommodationOfficer.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace HousingSearchListener.V1.Domain.Tenure
+{
+    public class TempAccommodationOfficer
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+    }
+}
```

Comment:
We probably need to add a ticket for Tech Debt to investigate why HSL is creating duplicate Domain models :thinking:

Reply:
I agree. We need to add id property to this object, so I'll sort that out in the shared and refactor this object out of the listener in the next PR unless I can see a good reason not to.